samples/excel/01-basics/multiple-property-set.yaml

```diff
@@ -0,0 +1,155 @@
+order: 3
+id: excel-multiple-property-set
+name: Multiple Property Set
+description: Setting multiple properties at once with the rich API object set() method.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#setup").click(setup);
+
+		$("#set-multiple-properties-with-object").click(setMultiplePropertiesWithObject);
+		$("#copy-properties-from-range").click(copyPropertiesFromRange);
+
+		async function setMultiplePropertiesWithObject() {
+			try {
+				await Excel.run(async (context) => {
+					const sheet = context.workbook.worksheets.getItem("Sample");
+
+					const range = sheet.getRange("B2:E2");
+					range.set({
+						format: {
```

Comment:
I just think it would be good to show that it's not just nested properties (format/font and format/fill) that you can set, but also a root-level property (like values)

Charts is another good object to do a "set" on, it has a bunch of properties at both the root level and deeper-nested (and "set" makes such API calls look much cleaner than having a wall of repeating text)